scipy/spatial/transform/rotation.py

```diff
@@ -1460,3 +1460,127 @@ def __getitem__(self, indexer):
                [ 0.57735027,  0.57735027, -0.57735027,  0.        ]])
         """
         return self.__class__(self._quat[indexer], normalized=True)
+
+
+class Slerp(object):
+    """Spherical Linear Interpolation of Rotations.
+
+    The interpolation between consecutive rotations is performed as a rotation
+    around a fixed axis with a constant angular velocity [1]_. This ensures
+    that the interpolated rotations follow the shortest path between initial
+    and final orientations.
+
+    Parameters
+    ----------
+    times : array_like, shape (N,)
+        Times of the known rotations. At least 2 times must be specified.
+    rotations : `Rotation` instance
+        Rotations to perform the interpolation between. Must contain N
+        rotations.
+
+    Methods
+    -------
+    __call__
+
+    References
+    ----------
+    .. [1] `Quaternion Slerp
+            <https://en.wikipedia.org/wiki/Slerp#Quaternion_Slerp>`_
+
+    Examples
+    --------
+    >>> from scipy.spatial.transform import Rotation as R
+    >>> from scipy.spatial.transform import Slerp as S
+
+    Setup the fixed keyframe rotations and times:
+
+    >>> key_rots = R.from_quat(np.random.uniform(size=(5, 4)))
+    >>> key_times = [0, 1, 2, 3, 4]
+
+    Create the interpolator object:
+
+    >>> interpolator = S(key_times, key_rots)
+
+    Interpolate the rotations at the given times:
+
+    >>> times = [0, 0.5, 0.25, 1, 1.5, 2, 2.75, 3, 3.25, 3.60, 4]
+    >>> interp_rots = interpolator(times)
+
+    The interpolator returns the same rotations at the keyframe times:
+
+    >>> key_rots.as_quat()
```

Comment:
Probably use Euler angles (in degrees) to demonstrate that this indeed looks like interpolation.